config.md

```diff
@@ -356,7 +356,8 @@ Runtime implementations MAY support any valid values for platform-specific field
 
 ## <a name="configHooks" />Hooks
 
-Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container.
+Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container if supported by the platform.
```

Comment:
If we *don't* clearly say which platforms can rely on `hooks`, we're back to the rejected #472.